Sources/Prometheus/Prometheus.swift

```diff
@@ -199,30 +234,17 @@ public class PrometheusClient {
         forType type: T.Type,
         named name: String,
         helpText: String? = nil,
-        quantiles: [Double] = defaultQuantiles) -> Summary<T, EmptySummaryLabels>
+        quantiles: [Double] = defaultQuantiles) -> PromSummary<T, EmptySummaryLabels>
     {
         return self.createSummary(forType: type, named: name, helpText: helpText, quantiles: quantiles, labels: EmptySummaryLabels.self)
     }
-    
-    // MARK: - Info
-    
-    /// Creates an Info metric with the given values
-    ///
-    /// - Parameters:
-    ///     - name: Name of the info
-    ///     - helpText: Help text for the info. Usually a short description
-    ///     - labelType: Type of labels this Info can use
-    ///
-    /// - Returns Info instance
-    public func createInfo<U: MetricLabels>(
-        named name: String,
-        helpText: String? = nil,
-        labelType: U.Type) -> Info<U>
-    {
-        let info = Info<U>(name, helpText, self)
-        prometheusQueue.async(flags: .barrier) {
-            self.metrics.append(info)
-        }
-        return info
-    }
 }
+
+/// Prometheus specific errors
+public enum PrometheusError: Error {
+    /// Thrown when a user tries to retrive
+    /// a `PromtheusClient` from `MetricsSystem`
+    /// but there was no `PrometheusClient` bootstrapped
+    case PrometheusFactoryNotBootstrapped
```

Comment:
AFAIK style is to have those lower case, or am I wrong (I totally could be wrong here)?

Reply:
you're right (since Swift 3 or so they're lower-case)

Reply:
Updated 👌